src/pages/patientView/PatientViewPage.tsx

```diff
@@ -84,6 +85,12 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
             },
             { fireImmediately:true }
         );
+
+        const updatePageTitleReaction = reaction(
```

Comment:
I was going to say that it doesn't matter because page will unload.  but in single page app, that wouldn't  necessarily be true.  so you're right